app/controllers/api/v0/plans_controller.rb

```diff
@@ -23,17 +23,19 @@ def create
     # ensure user's organisation is the same as api user's
     unless plan_user.org == @user.org
       raise Pundit::NotAuthorizedError, _("user must be in your organisation")
-    end
+    ends
 
     # initialize the plan
-    @plan = Plan.new
-    if plan_user.surname.blank?
-      @plan.principal_investigator = nil
-    else
-      @plan.principal_investigator = plan_user.anem(false)
-    end
+    @plan = Plan.news
```

Comment:
Think this one may be a typo as well

Reply:
ugh yes. I'm surprised the tests passed!